packages/core/src/editor/query.tsx

```diff
@@ -44,6 +44,8 @@ export function QueryMethods(state: EditorState) {
   return {
     /**
      * Determine the best possible location to drop the source Node relative to the target Node
+     *
+     * TODO: replace with Positioner.computeIndicator();
      */
     getDropPlaceholder: (
       source: NodeSelector,
```

Comment:
Can't replace this right now since `Positioner` requires the editor `store` as an input. Something we could look at once we merge the refactored editor store (#259) into this branch

Reply:
HMM :laughing: